src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php

```diff
@@ -159,18 +159,20 @@ private function connect()
             }
         }
 
-        $this->connection = ldap_connect($this->config['connection_string']);
+        $connectResult = ldap_connect($this->config['connection_string']);
+
+        if (false === $connectResult) {
```

Comment:
should be merged with the previous line `if (false === $connection = ldap_connect[...]`